react/components/EXPERIMENTAL_Table/README.md

```diff
@@ -642,6 +642,1091 @@ function VisibilityExample() {
+    (group, amt) => {
+      const calcDiscount = discountCurry(amt)
+      const update = bulkUpdate(group)
+      update(item => ({
+        ...item,
+        retailPrice: calcDiscount(item.retailPrice),
+        wholesalePrice: calcDiscount(item.wholesalePrice),
+      }))
+    },
+    [items]
+  )
+
+  const increaseQty = React.useCallback(
+    (group, amt) => {
+      const update = bulkUpdate(group)
+      update(item => ({
+        ...item,
+        qty: item.qty + amt,
+      }))
+    },
+    [items]
+  )
+
+  const decreaseQty = React.useCallback(
+    (group, amt) => {
+      const update = bulkUpdate(group)
+      update(item => ({
+        ...item,
+        qty: item.qty - amt,
+      }))
+    },
+    [items]
+  )
+
+  return { items, applyDiscount, increaseQty, decreaseQty }
+}
+
+function BulkFullExample() {
+  const { items, applyDiscount, increaseQty, decreaseQty } = useProducts()
+
+  const primaryAction = {
+    label: 'Apply 50% Discount',
+    onClick: () => applyDiscount(checkboxes.checkedItems, 0.5),
+  }
+
+  const secondaryActions = {
+    label: 'Quantity',
+    actions: [
+      {
+        label: 'Increase 50',
+        onClick: checked => increaseQty(checked, 50),
+      },
+      {
+        label: 'Decrease 50',
+        onClick: checked => decreaseQty(checked, 50),
+      },
+    ],
+    onActionClick: action => action.onClick(checkboxes.checkedItems),
+  }
+
+  const measures = useTableMeasures({
+    size: items.length,
+  })
+
+  const checkboxes = useCheckboxTree({
+    items,
+    onToggle: ({ checkedItems }) => console.table(checkedItems),
+  })
+
+  return (
+    <>
+      <Table
+        measures={measures}
+        checkboxes={checkboxes}
+        columns={columns}
+        items={items}>
+        <Table.Bulk active={checkboxes.someChecked}>
+          <Table.Bulk.Actions>
+            <Table.Bulk.Actions.Primary {...primaryAction} />
+            <Table.Bulk.Actions.Secondary {...secondaryActions} />
+          </Table.Bulk.Actions>
+          <Table.Bulk.Tail>
+            {!checkboxes.allChecked && (
+              <Table.Bulk.Tail.Info>
+                All rows selected: {checkboxes.checkedItems.length}
+              </Table.Bulk.Tail.Info>
+            )}
+            <Table.Bulk.Tail.Toggle
+              button={{
+                text: `Select all ${items.length}`,
+                onClick: checkboxes.checkAll,
+              }}
+              active={checkboxes.allChecked}>
+              Selected rows: {items.length}
+            </Table.Bulk.Tail.Toggle>
+            <Table.Bulk.Tail.Dismiss onClick={checkboxes.uncheckAll} />
+          </Table.Bulk.Tail>
+        </Table.Bulk>
+      </Table>
+    </>
+  )
+}
+
+;<BulkFullExample />
+```
+
+## Autocomplete, Filters & Pagination
+
+These components are grouped on one single example because they are often seen together on a real app. Each one has it's own documentation, so it's important to you check it out to see the full capabilities.
```

Comment:
I think we don't need to expose that the real app is grouped by these components. If someone uses without one of them, the person building the app is working in a toy project? I think it is not true and can be a little arrogant. Let's avoid this! :D